tools/ipapy2/ipapy2_isotope_patterns.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+import sys
+import pandas as pd
+from ipaPy2 import ipa
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(description="mapping the isotope pattern within the dataset.")
+    parser.add_argument("input_filename", type=str, help="a dataframe of clustered features.")
+    parser.add_argument("--isotope_diff", type=float, help="Difference between isotopes of charge 1.")
+    parser.add_argument("--ppm", type=float, help="Default value 100. Maximum ppm value allowed between 2 isotopes.")
+    parser.add_argument("--ionisation", type=int, help="ionisation. +1 or -1.")
+    parser.add_argument("--isotope_ratio", type=float, defaul=1, help="mininum intensity ratio expressed (Default value 0.5%). Only isotopes with intensity higher than this value/100 of the main isotope are considered.")
```

Comment:
This is not really the ratio tolerance more the minimum intensity, so an isotope intensity cutoff.